docs/examples.rst

```diff
@@ -53,29 +48,32 @@ Example: synthesizing 1000 soundscapes in one go
     time_stretch_min = 0.8
     time_stretch_max = 1.2
 
+    # generate a random seed for this Scaper object
+    seed = np.random.randint(0, 100000)
+
+    ## alternate ways to define random state:
+    #     seed = np.random.RandomState(0)
```

Comment:
To keep consistent with the previous example, `seed = np.random.RandomState(123)`.